all-of-us/workbench
# Chavanr/e2e cohort review participant detail

added the following steps to the review the participant in detail:
1. create new annotations field: type - free text
2. verify if the same field displays for other participants in the review sets

Pending:
1. create other annotation field types: numeric field /Date/dropdown
2. Edit  annotation field names
3. Delete  annotation field

---
**PR checklist**

- [ ] I have run the E2E tests on ths change against my local UI + API server with `yarn test-local`
- [ ] If this includes a UI change, I have taken screen recordings or screenshots of the new behavior and notified the PO and UX designer
